compiler/src/codegen/compcore.re

```diff
@@ -1633,10 +1633,7 @@ let call_lambda = (~tail=false, wasm_mod, env, func, (argsty, retty), args) => {
   );
 };
 
-let allocate_string = (wasm_mod, env, str) => {
-  let buf = Buffer.create(80);
-  Buffer.add_string(buf, str);
-
+let allocate_bytes_from_buffer = (wasm_mod, env, buf, tag, label) => {
```

Comment:
```suggestion
let allocate_byte_like_from_buffer = (wasm_mod, env, buf, tag, label) => {
```

Reply:
I think this name might be a little more clear, since this is being used by both byte and string allocations

Reply:
That's a good call!